api/v1alpha/src/admin/bq/procedure/permissionsDiff.sql

```diff
@@ -43,6 +43,7 @@ policyData AS (
   LEFT JOIN UNNEST(r.datasets) d
   LEFT JOIN UNNEST(d.tables) t
   LEFT JOIN userPolicies p ON r.policyId = p.policyId
+  WHERE d.datasetId IS NOT NULL
```

Comment:
To force the procedure to update for now, we can add documentation into the update steps to click the 'Initialize Schema' button after performing the upgrade. Or, we can create a separate button to 'Upgrade Schema', and we can build a separate mechanism to track and apply updates - but we probably need a separate effort for that.

Reply:
ok. Let's add a quick note in the README for now and cut a feature request for a more comprehensive upgrade strategy.